src/redux/orderSlice.js

```diff
@@ -43,11 +42,13 @@ export const fetchCreateOrder = createAsyncThunk(
 
 export const fetchUpdateOrder = createAsyncThunk(
   'order/updateOrder',
-  async ({ id, couponCode }) => {
-    const {
-      responseData: { order }
-    } = await updateOrder(id, { couponCode });
-    return order;
+  async ({ id, couponCode }, { rejectWithValue }) => {
+    try {
+      const { order } = await updateOrder(id, { couponCode });
```

Comment:
something is wrong with the update order error handling 
![Screen Shot 2022-12-12 at 9 43 56 AM](https://user-images.githubusercontent.com/41182484/207000662-68963e05-2b99-41b7-8860-711fbc4fb48a.png)